src/render/SDL_render.c

```diff
@@ -4150,6 +4151,7 @@ static int SDLCALL SDL_SW_RenderGeometryRaw(SDL_Renderer *renderer,
 
     return retval;
 }
+#endif
```

Comment:
Can you add `/* SDL_VIDEO_RENDER_SW */` here so someone looking at the code knows what the `#endif` corresponds to?